Tools/TestWebKitAPI/glib/CMakeLists.txt

```diff
@@ -180,6 +180,7 @@ ADD_WK2_TEST(TestWebKitSecurityOrigin ${TOOLS_DIR}/TestWebKitAPI/Tests/WebKitGLi
 ADD_WK2_TEST(TestWebKitSettings ${TOOLS_DIR}/TestWebKitAPI/Tests/WebKitGLib/TestWebKitSettings.cpp)
 ADD_WK2_TEST(TestWebKitURIUtilities ${TOOLS_DIR}/TestWebKitAPI/Tests/WebKitGLib/TestWebKitURIUtilities.cpp)
 ADD_WK2_TEST(TestWebKitWebContext ${TOOLS_DIR}/TestWebKitAPI/Tests/WebKitGLib/TestWebKitWebContext.cpp)
+ADD_WK2_TEST(TestWebKitWebExtensionMatchPattern ${TOOLS_DIR}/TestWebKitAPI/Tests/WebKitGLib/TestWebKitWebExtensionMatchPattern.cpp)
```

Comment:
This broke WPE:

```
[2068/2251] Linking CXX executable bin/TestWebKitAPI/WPE/TestWebKitWebExtensionMatchPattern
FAILED: bin/TestWebKitAPI/WPE/TestWebKitWebExtensionMatchPattern 
: && /usr/bin/c++ -DASSERT_ENABLED=1 -fdiagnostics-color=always -Wextra -Wall -pipe -fmax-errors=20 -Wno-odr -Wno-stringop-overread -Wno-stringop-overflow -Wno-nonnull -Wno-array-bounds -Wno-expansion-to-defined -Wno-noexcept-type -Wno-psabi -Wno-misleading-indentation -Wno-maybe-uninitialized -Wundef -Wpointer-arith -Wmissing-format-attribute -Wformat-security -Wcast-align -Wno-tautological-compare -fasynchronous-unwind-tables -fno-strict-aliasing -fno-exceptions -fno-rtti -fcoroutines -ffunction-sections -fdata-sections -O3 -DNDEBUG -fuse-ld=lld -Wl,--gc-sections -Wl,--disable-new-dtags Tools/TestWebKitAPI/glib/CMakeFiles/TestWebKitWebExtensionMatchPattern.dir/__/Tests/WebKitGLib/TestWebKitWebExtensionMatchPattern.cpp.o -o bin/TestWebKitAPI/WPE/TestWebKitWebExtensionMatchPattern  -Wl,-rpath,/app/webkit/WebKitBuild/WPE/Release/lib  /usr/lib/x86_64-linux-gnu/libglib-2.0.so  /usr/lib/x86_64-linux-gnu/libsoup-3.0.so  lib/libWebKitGLibAPITestsCore.a  lib/libWPEWebKit-2.0.so.2.4.1  /usr/lib/x86_64-linux-gnu/libseccomp.so  /usr/lib/x86_64-linux-gnu/libicudata.so  /usr/lib/x86_64-linux-gnu/libicui18n.so  /usr/lib/x86_64-linux-gnu/libicuuc.so  /usr/lib/x86_64-linux-gnu/libsystemd.so  /usr/lib/x86_64-linux-gnu/libbacktrace.a  /usr/lib/x86_64-linux-gnu/libsysprof-capture-4.a  /usr/lib/x86_64-linux-gnu/libxml2.so  /usr/lib/x86_64-linux-gnu/libsqlite3.so  /usr/lib/x86_64-linux-gnu/libxslt.so  lib/libGLESv2.a  lib/libANGLE.a  /usr/lib/x86_64-linux-gnu/liblcms2.so  /usr/lib/x86_64-linux-gnu/libwoff2dec.so  lib/libxdgmime.a  /usr/lib/x86_64-linux-gnu/libharfbuzz-icu.so  /usr/lib/x86_64-linux-gnu/libgcrypt.so  /usr/lib/x86_64-linux-gnu/libgpg-error.so  /usr/lib/x86_64-linux-gnu/libcrypto.so  /usr/lib/x86_64-linux-gnu/libwebpdemux.so  /usr/lib/x86_64-linux-gnu/libjxl.so  /usr/lib/x86_64-linux-gnu/libavif.so  -latomic  /usr/lib/x86_64-linux-gnu/libgio-2.0.so  /usr/lib/x86_64-linux-gnu/libgstallocators-1.0.so  /usr/lib/x86_64-linux-gnu/libgstapp-1.0.so  /usr/lib/x86_64-linux-gnu/libgstbase-1.0.so  /usr/lib/x86_64-linux-gnu/libgstpbutils-1.0.so  /usr/lib/x86_64-linux-gnu/libgstaudio-1.0.so  /usr/lib/x86_64-linux-gnu/libgsttranscoder-1.0.so  /usr/lib/x86_64-linux-gnu/libgstgl-1.0.so  /usr/lib/x86_64-linux-gnu/libgsttag-1.0.so  /usr/lib/x86_64-linux-gnu/libgstvideo-1.0.so  /usr/lib/x86_64-linux-gnu/libgstrtp-1.0.so  /usr/lib/x86_64-linux-gnu/libgstsdp-1.0.so  /usr/lib/x86_64-linux-gnu/libgstwebrtc-1.0.so  /usr/lib/x86_64-linux-gnu/libgstfft-1.0.so  /usr/lib/x86_64-linux-gnu/libocdm.so  /usr/lib/x86_64-linux-gnu/libflite.so  /usr/lib/x86_64-linux-gnu/libflite_usenglish.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_grapheme_lang.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_grapheme_lex.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_indic_lang.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_indic_lex.so  /usr/lib/x86_64-linux-gnu/libflite_cmulex.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_time_awb.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_awb.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_kal16.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_kal.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_rms.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_slt.so  /usr/lib/x86_64-linux-gnu/libtasn1.so  /usr/lib/x86_64-linux-gnu/libopenxr_loader.so  -latomic  /usr/lib/x86_64-linux-gnu/libgio-2.0.so  /usr/lib/x86_64-linux-gnu/libgstallocators-1.0.so  /usr/lib/x86_64-linux-gnu/libgstapp-1.0.so  /usr/lib/x86_64-linux-gnu/libgstbase-1.0.so  /usr/lib/x86_64-linux-gnu/libgstpbutils-1.0.so  /usr/lib/x86_64-linux-gnu/libgstaudio-1.0.so  /usr/lib/x86_64-linux-gnu/libgsttranscoder-1.0.so  /usr/lib/x86_64-linux-gnu/libgstgl-1.0.so  /usr/lib/x86_64-linux-gnu/libgsttag-1.0.so  /usr/lib/x86_64-linux-gnu/libgstvideo-1.0.so  /usr/lib/x86_64-linux-gnu/libgstrtp-1.0.so  /usr/lib/x86_64-linux-gnu/libgstsdp-1.0.so  /usr/lib/x86_64-linux-gnu/libgstwebrtc-1.0.so  /usr/lib/x86_64-linux-gnu/libgstfft-1.0.so  /usr/lib/x86_64-linux-gnu/libocdm.so  /usr/lib/x86_64-linux-gnu/libflite.so  /usr/lib/x86_64-linux-gnu/libflite_usenglish.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_grapheme_lang.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_grapheme_lex.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_indic_lang.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_indic_lex.so  /usr/lib/x86_64-linux-gnu/libflite_cmulex.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_time_awb.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_awb.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_kal16.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_kal.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_rms.so  /usr/lib/x86_64-linux-gnu/libflite_cmu_us_slt.so  /usr/lib/x86_64-linux-gnu/libtasn1.so  /usr/lib/x86_64-linux-gnu/libopenxr_loader.so  /usr/lib/x86_64-linux-gnu/libgmodule-2.0.so  /usr/lib/x86_64-linux-gnu/libsoup-3.0.so  /usr/lib/x86_64-linux-gnu/libgstreamer-1.0.so  -ldl  /usr/lib/x86_64-linux-gnu/libgbm.so  /usr/lib/x86_64-linux-gnu/libdrm.so  lib/libWPEToolingBackends.a  /usr/lib/x86_64-linux-gnu/libgobject-2.0.so  /usr/lib/x86_64-linux-gnu/libglib-2.0.so  lib/libSkia.a  /usr/lib/x86_64-linux-gnu/libepoxy.so  /usr/lib/x86_64-linux-gnu/libharfbuzz.so  /usr/lib/x86_64-linux-gnu/libjpeg.so  /usr/lib/x86_64-linux-gnu/libpng.so  /usr/lib/x86_64-linux-gnu/libz.so  /usr/lib/x86_64-linux-gnu/libfontconfig.so  /usr/lib/x86_64-linux-gnu/libfreetype.so  /usr/lib/x86_64-linux-gnu/libexpat.so  /usr/lib/x86_64-linux-gnu/libwebpmux.so  /usr/lib/x86_64-linux-gnu/libwebp.so  /usr/lib/x86_64-linux-gnu/libwpe-1.0.so  /usr/lib/x86_64-linux-gnu/libatk-bridge-2.0.so  /usr/lib/x86_64-linux-gnu/libWPEBackend-fdo-1.0.so  -lxkbcommon  -lwayland-server  -lwayland-egl  -lwayland-client  /usr/lib/x86_64-linux-gnu/libatk-1.0.so  -Wl,-rpath-link,/app/webkit/WebKitBuild/WPE/Release/lib && :
ld.lld: error: undefined symbol: beforeAll()
>>> referenced by TestMain.cpp
>>>               ../Tools/TestWebKitAPI/glib/CMakeFiles/WebKitGLibAPITestsCore.dir/WebKitGLib/TestMain.cpp.o:(main) in archive lib/libWebKitGLibAPITestsCore.a
ld.lld: error: undefined symbol: afterAll()
>>> referenced by TestMain.cpp
>>>               ../Tools/TestWebKitAPI/glib/CMakeFiles/WebKitGLibAPITestsCore.dir/WebKitGLib/TestMain.cpp.o:(main) in archive lib/libWebKitGLibAPITestsCore.a
collect2: error: ld returned 1 exit status
```

Problem is `ENABLE(WK_WEB_EXTENSIONS)` is not defined. Two options to handle it: (a) always define `beforeAll()` and `afterAll()`, (b) check `ENABLE_WK_WEB_EXTENSIONS` here in the CMake rules to build the entire test conditionally, as is done for TestWebKitNetworkSession just below.